docs/documentation/upgrading/topics/changes/changes-26_1_0.adoc

```diff
@@ -44,6 +44,11 @@ The following APIs for the JavaScript Authorization client are deprecated and wi
 
 These APIs are no longer needed as initialization is done automatically on demand when calling methods on the `KeycloakAuthorization` instance. You can safely remove any code that depends on these APIs.
 
+=== Deprecated endpoint for initiate registration from OIDC client
+
+The endpoint `/realms/<realm>/protocol/openid-connect/registrations` used for initiating registration by OIDC client is now deprecated as there is standard way for initiating registration from OIDC client,
+which is now supported by {project_name}. The standard way uses the parameter `prompt=create`.
```

Comment:
```suggestion
The `/realms/<realm>/protocol/openid-connect/registrations` endpoint, which was used for initiating registration by OIDC client, is now deprecated because standard way now exists to initiate registration from the OIDC client,
which is now supported by {project_name}. The standard way uses the parameter `prompt=create`.
```